app/javascript/legacy_react/src/components/common/layout.tsx

```diff
@@ -10,11 +10,11 @@ function arrayify<T>(items: T[]|T){
 
 type ClassNameableChildren = React.ReactElement<ClassNameable>[]|React.ReactElement<ClassNameable>
 
-export const TwoColumnFields: React.StatelessComponent<{children:ClassNameableChildren}> = (props) => {
+export const TwoColumnFields: React.FunctionComponent<{children:ClassNameableChildren}> = (props) => {
```

Comment:
You can just change this to a normal function. Look into function components on the React website.